liquidctl/driver/hydro_platinum.py

```diff
@@ -175,6 +186,15 @@ def get_status(self, **kwargs):
         Returns a list of `(property, value, unit)` tuples.
         """
 
+
+        d1 = bytes.fromhex("0101ffffffffffffffffffffffffff7f7f7f7fff00ffffffff00ffffffff00ffffffff00ffffffff00ffffffff00ffffffffffffffffffffffffffffff")
```

Comment:
Is this necessary?  By the way, `get_status` needs to be as lean as possible, since it may be used for device monitoring and maybe be feeding into programs that iteratively call it every 1 or 2 seconds.

Reply:
oops this is in the wrong spot, this should not be here. I didn't notice that the file stayed when I switched branches. Ill fix that.